server/parsec/components/invite.py

```diff
@@ -77,6 +77,7 @@ class ConduitListenCtx:
     state: ConduitState
     payload: bytes
     peer_payload: bytes | None
+    last_exchange: bool
```

Comment:
I'd name it `is_last_exchange` to remain consistent with other boolean flags.

Reply:
[195d06b](https://github.com/Scille/parsec-cloud/pull/7307/commits/195d06bc98e169d4a6f8583ab5f38c1bd1ef9de6)